microsoft-intune/microsoft_api_auth.py

```diff
@@ -0,0 +1,193 @@
+""" Copyright start
+  Copyright (C) 2008 - 2022 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end """
+
+from requests import request
+from time import time, ctime
+from datetime import datetime
+from connectors.core.connector import get_logger, ConnectorError
+
+CONFIG_SUPPORTS_TOKEN = True
```

Comment:
All imports at one place and all consts should be at one place for better code readability